### PR TITLE
Reorganized call feedback page

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
## Proposed changes
Reorganized call feedback page
https://user-images.githubusercontent.com/12942772/139911189-2136f4f6-d9e1-4828-8a23-e840e3d676c9.mov

[sc-1351]

### CODE REVIEW

File: src/components/Typography/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+interface Props {
+  className?: string;
+  children?: React.ReactNode | React.ReactNode[];
+  bold?: boolean;
+}
+
+export const H1 = ({ className, children, bold }: Props) => {
```
Comment (gabrielsaruhashi) [inline src/components/Typography/index.tsx:9]: figured it was a good time to add this typography layer
will make it easier for us to make the app responsive to different tablet sizes in the future

File: src/pages/CallFeedback/index.tsx [hunk 2]
```diff
@@ -1,60 +1,36 @@
...
 import Loader from "src/components/Loader";
 import { logout } from "src/redux/modules/session";
 import { addMinutes } from "date-fns";
 import { rateCall } from "src/api/Call";
-import { selectAllCalls } from "src/redux/selectors";
 import { updateProfile } from "src/api/User";
+import { useIntakeForm } from "src/hooks/useIntakeForm";
 
-type TParams = { id: string };
+type TParams = { id: string; exitType: CallExitType };
```
Comment (gabrielsaruhashi) [inline src/pages/CallFeedback/index.tsx:19]: this is setup work for the next PR where we start tracking whether the other participant ever attempt to join.

The call page will set the url param according to the state of RoomClient right before the user leaves the room.

The alternative was to re-load the room client in this page, but that felt overkill for this use case

File: src/components/CallFeedback/CallFeedback.tsx [hunk 3]
```diff
@@ -1,282 +1,157 @@
...
-  markIntakeFormAsComplete: () => void;
+  onSubmitIntakeForm: () => void;
+  promptIntakeForm: boolean;
 }
 
+const FeedbackWrapper = ({
+  children,
+}: {
+  children: React.ReactNode[] | React.ReactNode;
+}) => (
```
Comment (cainlevy) [inline src/components/CallFeedback/CallFeedback.tsx:42]: the `React.FC` type will take care of declaring children if you like:

```suggestion
const FeedbackWrapper: React.FC = ({children}) => (
```

File: src/components/Typography/index.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+interface Props {
+  className?: string;
+  children?: React.ReactNode | React.ReactNode[];
```
Comment (cainlevy) [inline src/components/Typography/index.tsx:5]: i recommend React.FC

**Review by cainlevy — APPROVED** [review]
i didn't spend the time to figure out what was new vs refactored, so i just skimmed for obvious lint.

**Pull request merged**